controllers/glance_controller.go

```diff
@@ -919,7 +871,17 @@ func (r *GlanceReconciler) generateServiceConfig(
 		templateParameters["ImageCacheDir"] = glance.ImageCacheDir
 	}
 
-	customData := map[string]string{glance.CustomConfigFileName: instance.Spec.CustomServiceConfig}
+	var tlsCfg *tls.Service
+	for _, api := range instance.Spec.GlanceAPIs {
+		if api.TLS.CaBundleSecretName != "" {
+			tlsCfg = &tls.Service{}
+			break
+		}
+	}
+	customData := map[string]string{
+		glance.CustomConfigFileName: instance.Spec.CustomServiceConfig,
+		"my.cnf":                    db.GetDatabaseClientConfig(tlsCfg), //(mschuppert) for now just get the default my.cnf
```

Comment:
ack, so the whole thing here is that we can get `my.cnf` as part of an API provided by the mariadb db object. Then we can use this information to set the database connection string.